build/env.py

```diff
@@ -0,0 +1,98 @@
+import os
+import shutil
+import subprocess
+import sys
+import sysconfig
+import tempfile
+import types
+
+
+if False:  # TYPE_CHECKING  # pragma: no cover
+    from typing import Dict, Optional, Iterable, Type
```

Comment:
What do you mean? `# TYPE_CHECKING` will proceed with the `if` if type checking is active.

It is guarded here because `Type` isn't in 3.5.0 and 3.5.1.